internal/file/watch/watch_test.go

```diff
@@ -139,58 +150,48 @@ func Test_watch_init(t *testing.T) {
 		return nil
 	}
 	tests := []test{
-		// TODO test cases
-		/*
-		   {
-		       name: "test_case_1",
-		       fields: fields {
-		           w: nil,
-		           eg: nil,
-		           dirs: nil,
-		           mu: sync.RWMutex{},
-		           onChange: nil,
-		           onCreate: nil,
-		           onRename: nil,
-		           onDelete: nil,
-		           onWrite: nil,
-		           onChmod: nil,
-		           onError: nil,
-		       },
-		       want: want{},
-		       checkFunc: defaultCheckFunc,
-		   },
-		*/
-
-		// TODO test cases
-		/*
-		   func() test {
-		       return test {
-		           name: "test_case_2",
-		           fields: fields {
-		           w: nil,
-		           eg: nil,
-		           dirs: nil,
-		           mu: sync.RWMutex{},
-		           onChange: nil,
-		           onCreate: nil,
-		           onRename: nil,
-		           onDelete: nil,
-		           onWrite: nil,
-		           onChmod: nil,
-		           onError: nil,
-		           },
-		           want: want{},
-		           checkFunc: defaultCheckFunc,
-		       }
-		   }(),
-		*/
+		{
+			name: "returns (nil, error) when watcher.Add returns no such file or directory",
+			fields: fields{
+				dirs: map[string]struct{}{
+					"vald.go": struct{}{},
+				},
+			},
+			want: want{
+				err: syscall.Errno(0x2),
+			},
+		},
+
+		{
+			name: "returns (*watch, nil) when no error occurs internally",
+			fields: fields{
+				dirs: map[string]struct{}{
```

Comment:
please add multiple files test (with all exists file test and some non-exists file)